extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCRecentKioskExtractor.java

```diff
@@ -0,0 +1,79 @@
+package org.schabi.newpipe.extractor.services.media_ccc.extractors;
+
+import com.grack.nanojson.JsonObject;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.localization.DateWrapper;
+import org.schabi.newpipe.extractor.stream.StreamInfoItemExtractor;
+import org.schabi.newpipe.extractor.stream.StreamType;
+
+import javax.annotation.Nullable;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class MediaCCCRecentKioskExtractor implements StreamInfoItemExtractor {
+
+    private final JsonObject event;
+
+    public MediaCCCRecentKioskExtractor(final JsonObject event) {
+        this.event = event;
+    }
+
+    @Override
+    public String getName() throws ParsingException {
+        return event.getString("title");
+    }
+
+    @Override
+    public String getUrl() throws ParsingException {
+        return event.getString("frontend_link");
+    }
+
+    @Override
+    public String getThumbnailUrl() throws ParsingException {
+        return event.getString("thumb_url");
+    }
+
+    @Override
+    public StreamType getStreamType() throws ParsingException {
+        return StreamType.VIDEO_STREAM;
+    }
+
+    @Override
+    public boolean isAd() throws ParsingException {
+        return false;
+    }
+
+    @Override
+    public long getDuration() throws ParsingException {
+        return event.getInt("duration");
+    }
+
+    @Override
+    public long getViewCount() throws ParsingException {
+        return event.getInt("view_count");
+    }
+
+    @Override
+    public String getUploaderName() throws ParsingException {
+        return "";
```

Comment:
event.getArray("persons").getString(0)

Reply:
person contains people who participated in a talk / video. That's more like a tag.

The uploader is in our case the conference, but we only have the URL for that conference and making 100 requests to get conference names is a little out of scope.

Reply:
From the [readme](https://github.com/voc/voctoweb):
>  Every talk (alias **event**, in other systems also called lecture or session) is assigned to exactly one **conference** (e.g. the _congress_ or lecture series like _datengarten_ or _openchaos_) and consists of multiple files alias **recordings**. These files can be video or audio recordings of the talk in different formats and languages (live-translation), subtitle tracks as srt or slides as pdf.